include/git2/merge.h

```diff
@@ -181,8 +189,8 @@ typedef struct {
 	/** The file to favor in region conflicts. */
 	git_merge_file_favor_t favor;
 
-	/** see `git_merge_file_flags_t` above */
-	unsigned int flags;
+	/** see `git_merge_file_flag_t` above */
+	git_merge_file_flag_t flags;
```

Comment:
Yes, and this doesn't preclude that.  This is to match the style in the rest of the library where enum types are named in the singular, even when they will be combined.  For example:  `git_repository_open_flag_t`.